addons/pos_restaurant/static/src/js/multiprint.js

```diff
@@ -190,16 +190,16 @@ models.Order = models.Order.extend({
         this.trigger('change',this);
     },
     computeChanges: function(categories){
-        var current_res = this.build_line_resume();
-        var old_res     = this.saved_resume || {};
+        var curr_list   = Object.values(this.build_line_resume());
```

Comment:
I don't really understand this diff, is it just to perform the find? because the build_line_resume method had not been changed.